b2sdk/stream/base.py

```diff
@@ -0,0 +1,9 @@
+import io
+
+
+class ReadOnlyMixin(object):
```

Comment:
```suggestion
class ReadOnlyStreamMixin(object):
```
One day we can have something else that will be read only and the names will become confusing

Reply:
yes :)